targetplatform/EclipseSmartHomeMavenBuild.launch

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<launchConfiguration type="org.eclipse.m2e.Maven2LaunchConfigurationType">
+<booleanAttribute key="M2_DEBUG_OUTPUT" value="false"/>
+<stringAttribute key="M2_GOALS" value="clean install -DskipChecks=true"/>
```

Comment:
may we also skip test execution at this point? Otherwise the launch config might be used during development, then test execution is crucial (as is the static code analysis). I'm torn on this.

Reply:
> may we also skip test execution at this point?

Have a look at line 10:
```
<booleanAttribute key="M2_SKIP_TESTS" value="true"/>
```

That launch configuration should not be used during development.
If the IDE is opened, you should not use Maven because the IDE could change files while building.
The setup part does not use a parallel build so it is not a problem there.